src/js/components/Text/Text.js

```diff
@@ -70,10 +71,10 @@ const Text = forwardRef(
     const styledTextResult = (
       <StyledText
         as={!as && tag ? tag : as}
-        colorProp={color}
         aria-label={a11yTitle}
-        truncate={truncate}
-        size={size}
+        $colorProp={color}
+        $truncate={truncate}
+        $size={size}
```

Comment:
Can we alphabetize these while we are here

Reply:
Yes, good call

Reply:
Done.